README.md

```diff
@@ -2,13 +2,33 @@
 
 Deployment setup for the Ingestion Service on  [Kubernetes](https://kubernetes.io/) clusters.
 
-## Set up local environment
+## Repository structure
+- `/apps`
+	-  ingest "applications" - deployments that provide functionality to our end users
+- `/infra`
+	- deployments, roles, service accounts, stateful sets etc. that support ingest "applications". These include things like mongo, monitoring, neo4j etc.
+	- terraform configuration for provisioning an ingest cluster
+- `/jobs`
+	-  one off jobs that can be ran
+- `/cron-jobs`
+	- regular jobs that are scheduled
+- `/config`
+	- Global configuration files used to deploy to each environment
+
+## About the cluster configuration
+- Our cluster is deployed to Amazon EKS
+- Each cluster is named `ingest-eks-<ENV>` e.g. `ingest-eks-dev`
+- The type of node and number of nodes is configured in `config/environment_<ENV>`  via `TF_VAR_node_size` and `TF_VAR_node_count`
+- Everything is deployed under one namespace of the form `<ENV>-environment` (e.g. `dev-environment`)
+
+## Development setup
+### Local environment
 We have migrated to helm 3, make sure to install the correct package version (>=3) on your system.
-### Mac
+#### Mac
 1. `git clone <this-repo-url>`
-2. Install [terraform](https://www.terraform.io/intro/getting-started/install.html) 13.5:
+2. Install [terraform](https://www.terraform.io/intro/getting-started/install.html) 0.12.25:
```

Comment:
Why were you deprecated the terraform version from `0.13.5` to `0.12.25`?

Reply:
0.12.25 is the one that was actually used - dont know why it was in the readme as 0.13.5

Reply:
Ok, that is strange. Locally I used `0.13.5` till now, but I agree let's use the version that we used before to create the cluster.